setup.py

```diff
@@ -135,12 +135,12 @@ def run(self):
     'dataclasses;python_version<"3.7"',
```

Comment:
Should this also be removed?

Reply:
Yes that seems only logical. Looks like I overlooked that one.